### PR TITLE
Widgets landing page

### LINKED ISSUE #813: Landing page for widget documentation app
**Enhancement**

At the moment the landing page for the documentation application is a blank page, it would be better if there was a general way to navigate to the docs from the home page.

Also a link in the menu to go back to the home page.

### PR DESCRIPTION
**Pull request by JamesLMilner**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**

The goal here was to be able to link to all the Widgets. I added a basic search bar that allows users to find specific Widgets that they might be looking for.

Resolves #813 

Screenshot_20191118_111259
Screenshot_20191118_113505

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-kjjfsaf3z.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-fv76bq2y8.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-qaujsr5pf.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-me55ekpf4.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-rdudrbx49.now.sh

File: src/examples/src/LinkedCard.m.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	right: 0;
+	text-decoration: none;
+	display: flex;
+	transition: box-shadow 0.4s ease;
+	width: 100%;
+	z-index: 1;
+}
+
+.link:hover {
+	box-shadow: 0 1px 0px rgba(0, 0, 0, 0.05), 0 2px 15px rgba(0, 0, 0, 0.05), 0 28px 50px rgba(0, 0, 0, 0.16);
```
Comment (rorticus) [inline src/examples/src/LinkedCard.m.css]: Should these px values be in reference to grid size?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-pr9gr378v.now.sh

File: src/examples/src/LinkedCard.m.css (hunk 1 above)
Comment (JamesLMilner) [reply src/examples/src/LinkedCard.m.css]: I'm not sure that makes too much sense for `box-shadow` especially when the values are so specific? I might be wrong. Either way maybe it should be a CSS variable.

Comment (agubler) [reply src/examples/src/LinkedCard.m.css]: FYI I don't think we used any css variables when we set up the examples app.

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-h4kq1rod9.now.sh

**Pull request merged**